lib/matplotlib/tests/test_tightlayout.py

```diff
@@ -157,6 +158,15 @@ def test_tight_layout8():
     example_plot(ax, fontsize=24)
 
 
+@image_comparison(baseline_images=['tight_layout9'])
+def test_tight_layout9():
```

Comment:
Sorry, please ignore. I misread `test_tight_layout8` as `test_tight_layouts`. Really need to fix the default fonts on this computer...